docs/en/Audit-Logging.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: global
+title: Audit Logging
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports audit logging of user access to your data. With audit logging, system administrators can
+keep track of each user's access to data in Alluxio.
+
+## Enable Audit Logging
+To enable Alluxio audit logging, you need to set the JVM property. In `./conf/alluxio-env.sh`, add the following two lines:
+
+```bash
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logging.enabled=true"
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=${ALLUXIO_MASTER_AUDIT_LOGGER:-MASTER_AUDIT_LOGGER}"
```

Comment:
Is it possible to make this a default? Ideally the user would only need to set one property to enable audit logging

Reply:
I thought sometimes users will want to configure their own log appender.

Reply:
It's good for advanced users to have the option to configure their own log appender, but it's also good for normal users to only need to set one property.

Reply:
Done.